jetson/nucleo_bridge/README.md

```diff
@@ -0,0 +1,104 @@
+Message: [SAOpenLoopCmd.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/SAOpenLoopCmd.lcm) \
+Publisher: onboard/teleop \
+Subscriber: onboard/nucleo_bridge
+
+#### RA Closed Loop \[Subscriber\] "/ik_ra_control"
+Message: [ArmPosition.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/ArmPosition.lcm) \
+Publisher: onboard/kinematics \
+Subscriber: onboard/nucleo_bridge
+
+#### SA Closed Loop \[Subscriber\] "/sa_closedloop_cmd"
+Message: [SAClosedLoopCmd.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/SAClosedLoopCmd.lcm) \
+Publisher: onboard/kinematics \
+Subscriber: onboard/nucleo_bridge
+
+#### Gimbal Open Loop \[Subscriber\] "/gimbal_openloop_cmd"
+Message: [GimbalCmd.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/GimbalCmd.lcm) \
+Publisher: onboard/teleop \
+Subscriber: onboard/nucleo_bridge
+
+#### Hand Open Loop \[Subscriber\] "/hand_openloop_cmd"
+Message: [HandCmd.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/HandCmd.lcm) \
+Publisher: onboard/teleop \
+Subscriber: onboard/nucleo_bridge
+
+#### Foot Open Loop \[Subscriber\] "/foot_openloop_cmd"
+Message: [FootCmd.lcm](https://github.com/raytitan/mrover-workspace/blob/rnucleo/rover_msgs/FootCmd.lcm) \
+Publisher: onboard/teleop \
+Subscriber: onboard/nucleo_bridge
+
+#### RA Pos Data \[Publisher\] "/arm_posiiton"
+Message: [ArmPosition.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/ArmPosition.lcm) \
+Publisher: onboard/nucleo_bridge \
+Subscriber: onboard/kinematics
+
+#### SA Pos Data \[Publisher\] "/sa_pos_data"
+Message: [SAPosData.lcm](https://github.com/umrover/mrover-workspace/blob/master/rover_msgs/SAPosData.lcm) \
+Publisher: onboard/nucleo_bridge \
+Subscriber: onboard/kinematics
+
+### Usage
+
+To build nucleo_bridge use `$./jarvis build onboard/nucleo_bridge/ ` from the mrover-workspace directory.
+
+To run nucleo_bridge use `$./jarvis exec onboard/nucleo_bridge/ `
+from the mrover-workspace directory.
+
+After initializing the LCM bus, I2C bus, and virtual Controller objects, the CLI will only show errors, since printing output to the console is time expensive. A blank CLI is a good thing.
+
+To control the RA/SA through open-loop
+* Ensure onboard/teleop is running on the same platform
+* Ensure base_station/gui is running on the base station
+* Operate the joysticks with the RA/SA Task page open on the base station GUI
+
+To control the RA/SA through closed-loop
+* Ensure onboard/teleop is running on the same platform
+* Ensure onboard/kinematics is running on the same platform
+* Ensure base_station/kineval is running on the base station
+* Input commands into the arm control page on the base station GUI
+
+### Common Errors
+
+This routine typically only thows one type of error, when it has issues communicating with the Nucleo motor controllers. They will have the form "command failed on channel"

```

Comment:
'Command Failed on Channel' should be its own header

Reply:
fixed